src/module-commons-ui.d.ts

```diff
@@ -150,6 +150,10 @@ declare module '@gridsuite/commons-ui' {
         disabled?: boolean;
     }>;
 
+    export const CancelButton: FunctionComponent<{
+        onClick: () => void;
```

Comment:
Maybe import ButtonProps from MUI then define this as `FunctionComponent<ButtonProps>` and avoid redefine onClick types and all. Same for SubmitButton right here

Reply:
Done